simple/stats/data.py

```diff
@@ -31,6 +31,8 @@
 _PREDICATE_SUB_CLASS_OF = "subClassOf"
 _PREDICATE_OBSERVATION_DATE = "observationDate"
 _PREDICATE_LOCATION = "location"
+_PREDICATE_POPULATION_TYPE = "populationType"
+_PREDICATE_MEASURED_PROPERTY = "measuredProperty"
```

Comment:
I we should support statType too.  And everything else in https://github.com/datacommonsorg/data/blob/master/docs/representing_statistics.md#statisticalvariable ?

Reply:
SG. b/322909728